apps/odyssey-stats/src/widget/highlights.jsx

```diff
@@ -6,7 +8,57 @@ import useTopPostsQuery from '../hooks/use-top-posts-query';
 
 import './hightlights.scss';
 
-function TopColumn( { items, viewAllUrl, viewAllText, title, isLoading, className = null } ) {
+const HIGHLIGHT_ITEMS_LIMIT = 5;
+const postAndPageLink = ( baseUrl, siteId, postId ) => {
+	return `${ baseUrl }#!/stats/post/${ postId }/${ siteId }`;
+};
+
+function ItemWrapper( { odysseyStatsBaseUrl, siteId, isItemLink, item } ) {
+	const translate = useTranslate();
+
+	const renderedItem = (
+		<div>
+			<p>{ item.title }</p>
+			<span>
+				{ translate( '%(views)s Views', {
+					args: { views: formattedNumber( item.views ) },
+				} ) }
+			</span>
+		</div>
+	);
+
+	return isItemLink ? (
+		<a
+			href={ postAndPageLink( odysseyStatsBaseUrl, siteId, item.id ) }
+			rel="noopener noreferrer"
+			title={ translate( 'View highlight post or page', {
+				textOnly: true,
+				context: 'View highlight post or page',
+			} ) }
+			aria-label={ translate( 'View highlight post or page', {
+				textOnly: true,
+				context: 'View highlight post or page',
+			} ) }
```

Comment:
```suggestion
			title={ translate( 'View detailed stats for %(title)s', {
				args: {
					title: item.title
				},
				textOnly: true,
				comment: 'Text for anchor linking to a stats page for a given post/page',
			} ) }
```

No need to add a separate `aria-label` if a `title` is already provided.